Makefile.js

```diff
@@ -335,6 +335,9 @@ function generatePrerelease(prereleaseId) {
     }
 
     commitSiteToGit(`v${releaseInfo.version}`);
+
+    // push to the next branch to trigger docs deploy
+    exec("git push origin HEAD:next -f");
```

Comment:
`generateRelease`/`generatePrerelease` should just prepare everything locally, and then `publishRelease` in the next step publishes the package and updates repositories on GitHub, so this should be in `publishRelease`. We should update `publishRelease` anyway to avoid updating the `latest` branch:

https://github.com/eslint/eslint/blob/a4e89dbe85589dab982885872dc206e090c27b3c/Makefile.js#L345-L352

Reply:
(we use the same `function publishRelease` for prereleases, too)

Reply:
Agree. We probably need to read `release-info.json` inside of `publishRelease` to get the correct data.

Reply:
Thanks! Done.